### ISSUE TITLE
is there a way to remove black bars when concatenating?

### ISSUE THREAD

**Issue by mrzpzp** [Zulko/moviepy#1724]
is there a way to remove blackbars when concatenating different size videos without messing up their size ratio?

for example if I try to concatenate two videos, one with 500px width and other with 1000px width, the output video will have the same width as the 1000px video and that will cause  black bars when the first video plays because its width is 500px.

I tried resizing the first and second video or the output video but they don't seem to work.

**Comment by Noah134** [comment]
It was already answered [here](https://github.com/Zulko/moviepy/issues/663#issuecomment-395322381) .

**Comment by mrzpzp** [comment]
> Maybe [this](https://github.com/Zulko/moviepy/issues/663#issuecomment-395322381) helps.
yeah I did something similar, I used the average value of width/height ratio for every video and resized all of them to that resolution. Although I wish there was a way to just remove black bars.

**Comment by chrischma** [comment]
any news on this?

**Comment by pioneerHitesh** [comment]
@keikoro can you assign this to me?

**Comment by OsaAjani** [comment]
Thank you for your contributions and for reporting issues in this repository. With the release of v2, which introduces significant changes to the codebase and API, we’ve reviewed the backlog of open PRs and issues. Due to the length of the backlog and the likelihood that many of these are either fixed or no longer applicable, we’ve made the decision to close all previous PRs and issues.

If you believe that any of these are still relevant to the current version or if you'd like to reopen a related discussion, please feel free to create a new issue or pull request, referencing the old one.

Thank you for your understanding and continued support!
